examples/.gitignore

```diff
@@ -2,3 +2,4 @@
 .transformers-cache
 composer.lock
 vendor
+tmp
```

Comment:
instead of this, what about using `sys_get_temp_dir` as default?

Reply:
Well, thing is, this directory is only used for examples, if we want the developer who's using the examples to access the file, the `sys_get_temp_dir` is probably not the best fit 🤔